x-pack/plugins/reporting/export_types/printable_pdf/server/execute_job/compatibility_shim.js

```diff
@@ -5,10 +5,22 @@
  */
 
 import url from 'url';
+import cookie from 'cookie';
 import { getAbsoluteUrlFactory } from './get_absolute_url';
+import { cryptoFactory } from '../../../../server/lib/crypto';
 
 export function compatibilityShimFactory(server) {
   const getAbsoluteUrl = getAbsoluteUrlFactory(server);
+  const crypto = cryptoFactory(server);
+
+  const decryptJobHeaders = async (job) => {
+    try {
+      const decryptedHeaders = await crypto.decrypt(job.headers);
+      return decryptedHeaders;
+    } catch (err) {
```

Comment:
Is there any benefit to either logging or appending the original error? We're losing it right now